_posts/2024-05-01-april-2024-roundup.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: "What we've been reading in April (2024)"
+description:
+  Here are the articles, videos, and tools that we've been excited about in April 2024, including interesting video game projects, a new report on the state of IoT software, and a tech talk on the new hardware model from Zephyr.
```

Comment:
We want to limit the line length -- I'll look at adding that to a precommit step. Fine for now